src/test/java/org/opentripplanner/generate/doc/UpdaterConfigDocTest.java

```diff
@@ -23,6 +24,7 @@
 import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
 
 @OnlyIfDocsExist
+@Tag("docs")
```

Comment:
Move this to `@OnlyIfDocsExist`: 
```
@Tag("docs")
@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
@EnabledIf("org.opentripplanner.generate.doc.framework.DocsTestConstants#docsExistOrWarn")
public @interface OnlyIfDocsExist {
}
```

Reply:
Not sure if we should rename the `OnlyIfDocsExist` - since it does more than one thing:
 - Tag test with "docs"
 - Check if docs directory exist
 
 Maybe `GenerateDocs` is a better name.

Reply:
I can do this when this PR is merged.

Reply:
That would be great. Merging now.